docs/source/pillars/computing_technology.md

```diff
@@ -33,8 +33,8 @@ _The interfaces used for interacting with the TRE management system and the TRE
     In practice a web browser is the most common way to achieve this.
   - Recommended
 * - A TRE workspace should provide an environment familiar to the users of the TRE.
-  - This may be in the form of a virtual Windows or Linux desktops, web applications, or a terminal.
-    The use of custom developed TRE-specific software should be avoided when widely used open-source alternatives already exist.
+  - This may take the form of a virtual Windows or Linux desktops, non-desktop interfaces such as JupyterLab and other web applications, or a terminal.
+    Bespoke TRE-specific software should be avoided when widely used open-source alternatives already exist.
```

Comment:
```suggestion
    Bespoke TRE-specific software should be avoided when widely used alternatives already exist.
```

Reply:
I do think open-source is a good aim, but I'm not sure it's reasonable to require it here. Thoughts @JimMadge ?